src/elife_profile/modules/custom/elife_templates/templates/elife_video.tpl.php

```diff
@@ -8,8 +8,9 @@
  */
 ?>
 <video controls="controls" poster="<?php print $poster; ?>" preload="none">
```

Comment:
`preload="none"` would be better as `preload="metadata"`. This enables the browser to determine how long the video is, and so present that information to the user, without the user having to start the video to find out.